### PR TITLE
Enhancing OpenStackStateMapper test according new definitions

### PR DESCRIPTION
**Pull request by llvieira**
I don't think the comment `// test case: ...` is needed here.

### CODE REVIEW

File: src/main/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackStateMapper.java [hunk 1]
```diff
@@ -12,24 +12,24 @@
...
-    private static final String BACKING_UP_STATUS = "backing-up";
-    private static final String RESTORING_BACKUP_STATUS = "restoring-backup";
-    private static final String ERROR_BACKING_UP_STATUS = "error_backing-up";
-    private static final String ERROR_RESTORING_STATUS = "error_restoring";
-    private static final String ERROR_EXTENDING_STATUS = "error_extending";
-    private static final String DOWNLOADING_STATUS = "downloading";
-    private static final String UPLOADING_STATUS = "uploading";
-    private static final String RETYPING_STATUS = "retyping";
-    private static final String EXTENDING_STATUS = "extending";
+    public static final String CREATING_STATUS = "creating";
```
Comment (thiagomanel) [inline src/main/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackStateMapper.java]: @llvieira I'm not sure we need this to be public. If you are using only for tests, shouldn't be protected?

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackStateMapperTest.java [hunk 2]
```diff
@@ -9,342 +9,383 @@
 
     @Test
     public void testComputeActiveStatusToReady() {
+        // set up
         InstanceType instanceType = InstanceType.COMPUTE;
 
-        InstanceState instanceState1 = OpenStackStateMapper.map(instanceType, "active");
-        InstanceState instanceState2 = OpenStackStateMapper.map(instanceType, "ACTIVE");
+        // exercise
```
Comment (thiagomanel) [inline src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackStateMapperTest.java]: I think you are swapping, again, actual and expected values ....

File: src/main/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackStateMapper.java (hunk 1 above)
Comment (llvieira) [reply src/main/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackStateMapper.java]: I made it protected now.

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackStateMapperTest.java (hunk 2 above)
Comment (llvieira) [reply src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackStateMapperTest.java]: Fixed!

**Pull request merged**